### DISCUSSION TITLE
What causes IfcConvert error "Argument is not a list of entity instances" ?

### DISCUSSION THREAD

**Discussion by ArnaudBBRI** [IfcOpenShell/IfcOpenShell#4455] [Q&A] [answered]
Hello,

I have received an IFC file, perfectly readable with blenderBIM.
I can run ifcConvert to OBJ with no problem like this :

`.\IfcConvert.exe -y -j 16 --use-element-guids --y-up '.\AA.ifc' ./AA.obj`

But when I run it to produce an XML

`.\IfcConvert.exe -v '.\AA.ifc' ./AA.xml`

Then I have the following error message:

> IfcOpenShell IfcConvert v0.7.0-e508fb44 (OCC 7.5.3)
> Scanning file...
> Done scanning file
> Parsing input file took 0 seconds
> Writing XML output...
> 
> Log:
> [Error] [2024-03-21 15:04:00] Argument is not a list of entity instances

What does the error message mean, and how can I avoid it ?

Thanks a lot,
Arnaud

**Comment by aothms** [comment, marked as answer]
Can you try running the file in `python -m ifcopenshell.validate`? Likely it will point you to the issue in the model.

**Reply by ArnaudBBRI** [reply]
Ok, found it with that python command! 
Some non-optional empty attribute.

Thank you!
